src/collective/behavior/amp/validators.py

```diff
@@ -40,3 +40,15 @@ def is_valid_logo(value):
             u'Image should be no wider than 600px, and no taller than 60px.'))
 
     return True
+
+
+def is_valid_tags(value):
+    url = 'www.cartacapital.com.br/internacional/jean-luc-melenchon-o-esquerdista-que-sacode-a-campanha-presidencial-francesa/@@amp'
+    r = requests.get('https://amp.cloudflare.com/q/' + url)
+    validation = r.json()
+    if validation['valid']:
+        print url, 'is valid'
```

Comment:
the whole motive of the feature is to display the error, if we not print the error then should we write log to some external file?